web/src/app/admin/skills/skills.component.ts

```diff
@@ -1,14 +1,105 @@
-import {Component, OnInit} from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { SkillsService } from './shared/skills.serviceApp';
+import { SkillViewModel } from '../../webapi/models/skill-view-model';
 
 @Component({
   selector: 'skills',
-  templateUrl: 'skills.component.html'
+  templateUrl: 'skills.component.html',
+  styleUrls: ['./skills.component.css']
 })
 
 export class SkillsComponent implements OnInit {
-  constructor() {
-  }
 
+  skillsModel: SkillViewModel = {} as SkillViewModel;
+  skills: SkillViewModel[] = [];
+  skillsList: SkillViewModel[];
+
+  isCreate: boolean = false;
+  isUpdate: boolean = false;
```

Comment:
update and create without using variable on the basis of id.

Reply:
this one using only for changing panel name